settings/urls.py

```diff
@@ -1,7 +1,24 @@
 from django.contrib import admin
 from django.urls import path, include
-
+from drf_spectacular.views import (
+    SpectacularAPIView,
+    SpectacularRedocView,
+    SpectacularSwaggerView,
+)
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/v1/", include("rental.urls")),
+    # YOUR PATTERNS
+    path("api/schema/", SpectacularAPIView.as_view(), name="schema"),
```

Comment:
Esta ruta descarga un archivo `Proyect API.yaml` esperaba que me mostrara algunos datos del api

Reply:
El archivo YAML generado por drf-spectacular es un archivo de especificación OpenAPI que describe la api y sirve para construir la documentación en otros entornos independientes a nuestra proyecto como:

https://editor.swagger.io/

https://redocly.github.io/redoc/

https://learning.postman.com/docs/getting-started/importing-and-exporting/importing-from-swagger/